content/en/security/application_security/threats/add-user-info.md

```diff
@@ -763,7 +763,7 @@ The following modes are deprecated:
 
 ## Disabling automatic user activity event tracking 
 
-You can disable automated user activity detection through your [ASM service catalog][14] by changing the automatic instrumentation mode to `disabled` on the service you want to deactivate.
+You can disable automated user activity detection through your [ASM service catalog][14] by changing the automatic instrumentation mode to `disabled` on the service you want to deactivate. This requires [Remote Configuration][15] to be enabled, but you can otherwise set the environment variable `DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING_ENABLED` to `false` on your service and restart it. This must be set on the application hosting the Datadog Tracing Library, and not on the Datadog Agent.
```

Comment:
@Julio-Guerra Just FYI, the link for Remote Configuration is missing a destination link at the bottom of the file. 

Also, just to clarify, this is saying that you can set `DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING_ENABLED` to `false` to disable automated user activity detection? The only caveat is that Remote Configuration must be enabled in order for this to work?

Reply:
@michaelcretzman - Being that this is literally my first time reviewing a Github PR can you please update Julio's addition (green) to (1) make it read better and address drichards-87's abvoe comment and (2) address the fact that now the following is repeated twice in a row "set the environment variable `DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING_ENABLED` to `false` on your service and restart it."